### DISCUSSION TITLE
How to improve accuracy

### DISCUSSION THREAD

**Discussion by jscdefk** [deepmodeling/deepmd-kit#2586] [Q&A]
[param.txt](https://github.com/deepmodeling/deepmd-kit/files/11641630/param.txt)
Hello teacher, this is my run phase param file and single point calculation INCAR. After running with dpgen, I found that the energy accuracy in the lcurve file is only 10 ^ -2ev, which is completely below the level of 10 ^ -4. I don't know what parameters to adjust and write to improve the accuracy

**Comment by njzjz** [comment]
You can read some general suggestions we provide on the page below.
https://docs.deepmodeling.com/projects/deepmd/en/master/troubleshooting/precision.html

**Reply by jscdefk** [reply]
Thank  you
